executor/executor.c

```diff
@@ -482,12 +484,9 @@ static void kill_subprocess() {
     if (waitpid(pid, &status, 0) == -1) {
         log_printf(ERROR, "Wait failed for pid %d: %s", pid, strerror(errno));
     }
-    if (!WIFEXITED(status)) {
-        log_printf(ERROR, "Error when shutting down execution of mode %d", mode);
-    }
-    if (WIFSIGNALED(status)) {
-        log_printf(ERROR, "killed by signal %d\n", WTERMSIG(status));
-    }
+    int sig_number = WTERMSIG(status);
+    log_printf((sig_number == DEBUG_SIGNAL) ? DEBUG : ERROR, "Error when shutting down execution of mode %d", mode);
+    log_printf((sig_number == DEBUG_SIGNAL) ? DEBUG : ERROR, "killed by signal %d\n", sig_number);

```

Comment:
Instead of doing this, can you just put `exit(0)` at the beginning of `python_exit_handler`? and I guess comment the rest of the handler for now.

Reply:
yea. but just comment out the other code in `python_exit_handler` for now.